### DISCUSSION TITLE
s/6yac_lqw3jb0lzljtx4a2g

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#17] [Announcements]
# s/6yac_lqw3jb0lzljtx4a2g

Django+Ansible构建任务中心思路

https://blog.ops-coffee.cn/s/6yac_lqw3jb0lzljtx4a2g

**Comment by lipeibin** [comment]
可以看源码吗

**Comment by lipeibin** [comment]
Can I check the source code?&nbsp;Beg for an github address.
&nbsp;


李先生
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年6月29日(星期四) 下午4:05
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: [ops-coffee/blog] s/6yac_lqw3jb0lzljtx4a2g (Discussion #17)





 
s/6yac_lqw3jb0lzljtx4a2g
 
Django+Ansible构建任务中心思路
 
https://blog.ops-coffee.cn/s/6yac_lqw3jb0lzljtx4a2g

**Comment by cf1998** [comment]
又学到了
